securedrop/source_templates/index.html

```diff
@@ -74,7 +74,7 @@ <h2 class="welcome-text">
     reading the page. Hidden by default. #}
     <div class="bubble" id="security-setting-info" class="info">
       <ol>
-        <li>{{ gettext('Click the <img src="{icon}" alt="shield icon"> in the toolbar above').format(icon=url_for("static", filename="i/font-awesome/black/guard.svg"))  }}</li>
+        <li>{{ gettext('Click the <img src="{icon}" alt="&quot;Security Level&quot; button"> in the toolbar above').format(icon=url_for("static", filename="i/font-awesome/black/guard.svg"))  }}</li>
```

Comment:
Since I can't boldface this (very small amount of :-) ["[t]ext taken directly from a user interface"](https://docs.securedrop.org/en/stable/development/documentation_guidelines.html#glossary), I thought I'd quote it instead.  But I'm not set on this if you think that `Security Level button` is clear enough in context to be worth the simplicity.